Provide an optional resolvconf package with a symlink to resolvectl.

This allows removing resolvconf while maintaining compatibility with
scripts and programs that utilize it.

Index: systemd/debian/control
===================================================================
--- systemd.orig/debian/control
+++ systemd/debian/control
@@ -569,3 +569,14 @@ Description: systemd development files
  are different from the libsystemd's and libudev's pkg-config files, which can
  still be found in the respective dev packages, but instead provide data such as
  the installation directories for units, and more.
+
+Package: systemd-resolvconf
+Section: admin
+Architecture: amd64 arm64 i386
+Priority: optional
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         systemd
+Conflicts: resolvconf
+Provides: resolvconf
+Description: Symlinks the resolvconf command to resolvectl for compatibility.
Index: systemd/debian/systemd-resolvconf.links
===================================================================
--- /dev/null
+++ systemd/debian/systemd-resolvconf.links
@@ -0,0 +1 @@
+/usr/bin/resolvectl /usr/sbin/resolvconf
